tests/fixture_layouts/routes/async/redirect/_layout.tsx

```diff
@@ -0,0 +1,13 @@
+import { LayoutProps, RouteContext } from "$fresh/server.ts";
+
+export default async function AsyncSubLayout(
+  req: Request,
+  ctx: RouteContext,
+  { Component }: LayoutProps,
+) {
+  await new Promise((r) => setTimeout(r, 10));
```

Comment:
```suggestion
  await delay(10);
```
Nit. And other instances. Reference: https://deno.land/std@0.197.0/async/mod.ts?s=delay